<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>oficina</class>
 <widget class="QDialog" name="oficina">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboOrigenFilter">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>27</y>
     <width>135</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboDestinoFilter">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>27</y>
     <width>135</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelFechaFilter">
   <property name="geometry">
    <rect>
     <x>624</x>
     <y>30</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Fecha&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateFechaFilter">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>30</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="reservas">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>800</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelPlazas">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Plazas&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinPlazas">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReservar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>5</y>
      <width>115</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>RESERVAR</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="labelOrigenFilter">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Origen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDestinoFilter">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Destino&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkFecha">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>605</x>
     <y>30</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tablaNegos">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>800</width>
     <height>380</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashLine</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="labelEstado">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>470</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>6</red>
         <green>232</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>6</red>
         <green>232</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>127</red>
         <green>127</green>
         <blue>127</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkFecha</sender>
   <signal>clicked()</signal>
   <receiver>oficina</receiver>
   <slot>showNegos()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>614</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboOrigenFilter</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>oficina</receiver>
   <slot>showNegos()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboDestinoFilter</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>oficina</receiver>
   <slot>showNegos()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tablaNegos</sender>
   <signal>currentItemChanged(QTableWidgetItem*,QTableWidgetItem*)</signal>
   <receiver>oficina</receiver>
   <slot>activarReservas()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>250</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tablaNegos</sender>
   <signal>itemClicked(QTableWidgetItem*)</signal>
   <receiver>oficina</receiver>
   <slot>activarReservas()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnReservar</sender>
   <signal>clicked()</signal>
   <receiver>oficina</receiver>
   <slot>reservar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>456</x>
     <y>520</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dateFechaFilter</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>oficina</receiver>
   <slot>showNegos()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>721</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>718</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>activarFecha()</slot>
  <slot>showNegos()</slot>
  <slot>activarReservas()</slot>
  <slot>reservar()</slot>
 </slots>
</ui>
